HIP/hip-0000-nft-standard.md

```diff
@@ -0,0 +1,527 @@
+
+References: 
+https://github.com/hashgraph/did-method/blob/master/did-method-specification.md
+https://w3c.github.io/did-core/
+
+
+### description
+
+**Type:** string
+
+**Required**
+
+**Description:** Human-readable description of the NFT.
+
+
+### image
+
+**Type:** string (CID or URI)
+
+**Required**
+
+**Description:** CID or path to the NFT's **preview image** file. Each NFT should have a preview image set to allow NFT tooling to load a display image quickly. 
+
+For image-based NFTs, you can use a compressed image optimized for the web. For other types of NFTs, you can use any compressed image that gives a visual clue as to what to expect as a user. 
+
+It's recommended to host your file on [IPFS](https://ipfs.io/) and use a service like [Pinata](https://pinata.cloud/) to easily pin your file. Your CID should look like this: `ipfs://<hash>`.
+
+Alternatively, you can use [Arweave](https://www.arweave.org/), receiving a similar CID that looks like this: `ar://<hash>`.
+
+
+### image_integrity
+
+**Type:** string (SHA-256)
+
+**Optional (recommended)**
+
+**Description:** SHA-256 digest of the file pointed by the `image` property. The `image_integrity` property that contains a cryptographic hash of the representation of the resource the author expects to load. 
+
+For instance, an author may wish to load some image from a shared server. Specifying that the expected SHA-256 hash of https://example.com/image.jpeg is ba7816bf8f01cfea414140de5dae2223b00361a396177a9cb410ff61f20015ad means that the user agent can verify that the data it loads from that URL matches that expected hash before loading the NFT. This integrity verification significantly reduces the risk that an attacker can substitute malicious content.
+
+References: https://w3c.github.io/webappsec-subresource-integrity/
+
+
+### type
+
+**Type:** string (MIME type)
+
+**Required**
+
+**Description:** Mime formatting shall follow the following format: **type/subtype**. As a rule, mime types are all lowercase. However apps should be programmed to accept any case for robustness.
+
+A list of common mime types can be found here: https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types
+
+Note that mime types for directories are not uniformly defined. Some IPFS CIDs point to directories rather than files, so this type is useful. This standard shall define it using the format: text/directory.
+
+
+### format
+
+**Type:** string
+
+**Required**
+
+**Possible values:** 
+
+- `none`: When not following any standard (not recommneded)
+- `opensea`: When following the [OpenSea standard](https://docs.opensea.io/docs/metadata-standards)
+- `HIPXXX`: When following a standard defined by HIPs, e.g. `HIP412`
+
+**Description:** Describes the used standard to display NFT metadata. This property allows for faster parsing by NFT tooling because they don't have to figure out which standard they are dealing with. 
```

Comment:
I'd like to propose that we update this to have a reference to a published schema. This would let parsers very quickly identify not only the schema type but also the version of the schema being used over time as the standards evolve. I've added more examples in the main discussion here: https://github.com/hashgraph/hedera-improvement-proposal/discussions/413#discussioncomment-3589586

Reply:
@justynspooner I've added a JSON schema 👍